File: alpaul_gls/Content/Downloadable/PHAO2018092520181018171302/Business/Organismo.cs
using System.Collections.Generic;

namespace Business
{
    public class Organismo
    {
	    private static Data.Organismo obj = new Data.Organismo();

        public static int Add(Models.Organismo value)
        {
            return obj.Add(value);
        }

        public static int Edit(Models.Organismo value)
        {
            return obj.Edit(value);
        }

        public static int Delete(stirng value)
        {
            return obj.Delete(value);
        }

        public static Models.Organismo GetById(string ID)
        {
            return obj.GetById(ID);
        }

		public static Models.Organismo GetByGUID(string ID)
        {
            return obj.GetByGUID(ID);
        }

        public static List<Models.Organismo> GetAll()
        {
            return obj.GetAll();
        }
    }
}using System.Collections.Generic;

namespace Business
{
    public class Organismo
    {
	    private static Data.Organismo obj = new Data.Organismo();

        public static int Add(Models.Organismo value)
        {
            return obj.Add(value);
        }

        public static int Edit(Models.Organismo value)
        {
            return obj.Edit(value);
        }

        public static int Delete(stirng value)
        {
            return obj.Delete(value);
        }

        public static Models.Organismo GetById(string ID)
        {
            return obj.GetById(ID);
        }

		public static Models.Organismo GetByGUID(string ID)
        {
            return obj.GetByGUID(ID);
        }

        public static List<Models.Organismo> GetAll()
        {
            return obj.GetAll();
        }
    }
}

File: alpaul_gls/Content/Downloadable/PHAO2018092520181018161350/Business/Organismo.vb
Imports System.Collections.Generic

Public Class Organismo
    Public Shared Function Add(value As Models.Organismo) As Integer
        Return Data.Organismo.Add(value)
    End Function

    Public Shared Function Edit(value As Models.Organismo) As Integer
        Return Data.Organismo.Edit(value)
    End Function

    Public Shared Function Delete(value As String) As Integer
        Return Data.Organismo.Delete(value)
    End Function

    Public Shared Function GetById(value As String) As Models.Organismo
        Return Data.Organismo.GetById(value)
    End Function

    Public Shared Function GetByGUID(value As String) As Models.Organismo
        Return Data.Organismo.GetByGUID(value)
    End Function

    Public Shared Function GetAll() As List(Of Models.Organismo)
        Return Data.Organismo.GetAll()
    End Function
End Class